### PR TITLE
Fix project and press links, correct template file

### PR DESCRIPTION
**Pull request by coreycaitlin**
Fixes issue(s) #2276 and others

[CircleCI](https://circleci.com/gh/18F/18f.gsa.gov/tree/fix-links)

[:sunglasses: PREVIEW](https://federalist.18f.gov/preview/18F/18f.gsa.gov/fix-links/)

Changes proposed in this pull request:
- Correct the press link in the footer
- Correct project-page urls
- Add links to new project pages from What we deliver
- Update the example project template to reflect more detail

/cc @gemfarmer @elainekamlley

### CODE REVIEW

**Comment by coreycaitlin** [comment]
This is also a test to see if our circle tests keep failing — I made a couple small changes to hopefully prevent all the false positives that might be slowing things down.

File: _about/index.md [hunk 1]
```diff
@@ -57,12 +57,12 @@ We are a team of about 200 designers, software engineers, strategists, and produ
 
 Want to get to know our team a little better? These blog posts feature the voices of people from across 18F:
 
-- [We asked 100 of our coworkers why they joined 18F]({{ site.baseurl }}/2016/03/21/we-asked-100-of-our-coworkers-why-did-you-join-18f/)
-- [We asked over 100 of our coworkers: What was your path to 18F?]({{ site.baseurl }}/2016/03/22/what-was-your-path-to-18f/)
-- [We asked over 100 of our coworkers what they've loved working on?]({{ site.baseurl }}/2016/03/23/what-have-you-loved-working-on/)
-- [We asked everyone at 18F to reflect on the most meaningful project they worked on this year]({{ site.baseurl }}/2015/12/23/looking-back-2015-our-own-words/)
+- [We asked 100 of our coworkers why they joined 18F](https://18f.gsa.gov/2016/03/21/we-asked-100-of-our-coworkers-why-did-you-join-18f/)
```
Comment (gemfarmer) [inline _about/index.md]: Are these changes from site.baseurl to 18f.gsa.gov just for CircleCI?

Comment (coreycaitlin) [inline _about/index.md]: Yep — I'll change them back and see what happens.

**Comment by gemfarmer** [comment]
So I may have come up with a decent solution, inspired by Corey's replacement of the baseurl variables! Since we don't really need to be able to navigate the site with the accesslint build, we should be able to replace all baseurls with `https://federalist.18f.gov/preview/18F/18f.gsa.gov/dev/`. The only problem with this approach is that new pages wouldn't have a link on `dev`, so we would be getting false positives there

**Pull request merged**